src/nft/components/collection/CollectionStats.tsx

```diff
@@ -254,6 +263,9 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
   // round daily volume & floorPrice to 3 decimals or less
   const totalVolumeStr = ethNumberStandardFormatter(stats.stats?.total_volume)
   const floorPriceStr = ethNumberStandardFormatter(stats.floorPrice)
+  const floorChangeStr =
+    stats.stats && stats.stats.one_day_change ? Math.round(Math.abs(stats.stats.one_day_change) * 100) : 0
```

Comment:
should be `stats.stats.one_day_floor_change` not `one_day_change`. One day change refers to 1 day TVL % change

Reply:
changed it, but atm that field is missing from the response. Need to follow up with BE